### ISSUE TITLE
Fix RFC7230

### ISSUE THREAD

**Issue by BigBlueHat** [BigBlueHat/userinfo#13]
See: https://hypothes.is/a/ZIoKvGXDQpehbcAXc1Y5iw

**Comment by BigBlueHat** [comment]
@azaroth42 @csarven @melvincarvalho your thoughts on how best to approach fixing RFC7230's `userinfo` :bug: would be most helpful. :smile:

**Comment by azaroth42** [comment]
Hmm. The last paragraph of 2.7.1 is pretty explicit that userinfo is now an error in HTTP 1.1 URIs
[Semantic versioning? What's that?]

Looks like an uphill battle, I'm afraid. Not sure I have any good ideas :(

**Comment by BigBlueHat** [comment]
Yeah...looks like I should've gotten into specification discussions a bit earlier. :frowning: 

Well, despite the "overreach" I think the use case(s) are compelling enough to soldier on and see how far we get.

They're still valid URI's. It's just currently invalid for an HTTP to parse them into a header. Once there's a reasonable HTTP header to parse just the `user` part into, then maybe it'll go down easier. :wink:

We'll see. :eyeglasses:
